.github/shared/src/changed-files.js

```
@@ -45,102 +45,91 @@ export async function getChangedFiles(options = {}) {
-      }
+  // Log all changed files by categories
+  if (logger) {
+    logger.info("Categorized Changed Files:");
 
-      if (categorizedFiles.modifications.length > 0) {
-        logger.info(`  Modifications (${categorizedFiles.modifications.length}):`);
-        for (const file of categorizedFiles.modifications) {
-          logger.info(`    M ${file}`);
-        }
+    if (categorizedFiles.additions.length > 0) {
+      logger.info(`  Additions (${categorizedFiles.additions.length}):`);
+      for (const file of categorizedFiles.additions) {
+        logger.info(`    + ${file}`);
       }
+    }
 
-      if (categorizedFiles.deletions.length > 0) {
-        logger.info(`  Deletions (${categorizedFiles.deletions.length}):`);
-        for (const file of categorizedFiles.deletions) {
-          logger.info(`    - ${file}`);
-        }
+    if (categorizedFiles.modifications.length > 0) {
+      logger.info(`  Modifications (${categorizedFiles.modifications.length}):`);
+      for (const file of categorizedFiles.modifications) {
+        logger.info(`    M ${file}`);
       }
+    }
 
-      if (categorizedFiles.renames.length > 0) {
-        logger.info(`  Renames (${categorizedFiles.renames.length}):`);
-        for (const rename of categorizedFiles.renames) {
-          logger.info(`    R ${rename.from} -> ${rename.to}`);
-        }
+    if (categorizedFiles.deletions.length > 0) {
+      logger.info(`  Deletions (${categorizedFiles.deletions.length}):`);
+      for (const file of categorizedFiles.deletions) {
+        logger.info(`    - ${file}`);
       }
+    }
 
-      logger.info(`  Total: ${categorizedFiles.total} files`);
-      logger.info("");
+    if (categorizedFiles.renames.length > 0) {
+      logger.info(`  Renames (${categorizedFiles.renames.length}):`);
+      for (const rename of categorizedFiles.renames) {
+        logger.info(`    R ${rename.from} -> ${rename.to}`);
+      }
     }
 
-    return categorizedFiles;
-  } catch (error) {
```

Better to let error bubble than catch and return empty